### DISCUSSION TITLE
SO making FTP communications to multiple regional registries

### DISCUSSION THREAD

**Discussion by ahamza4500** [Security-Onion-Solutions/securityonion#8201] [Unsupported Versions] [answered]
Hey,

I've been using Security Onion for a little bit now and I've seen on two occasions this month (on Saturdays at ~23:30 PST) that it makes FTP calls to multiple IP registries. I've seen ARIN, APNIC, LACNIC, RIPE NCC, AFRINIC, and there were around 20 ish calls in the space of a minute. I also see traffic over high ports (> 49000) to the same IPs.

I did look up this issue and found [this](https://www.reddit.com/r/securityonion/comments/ftagdx/comment/fm6ummu/?utm_source=share&utm_medium=web2x&context=3) on reddit, is that the case here?

Is there a reason why it's doing this? We don't have the packets captured so I can't really look at them. Why use FTP?

Thanks.

**Comment by xfaith** [comment]
I know on 16.04 it used to go out via FTP to download IP Region codes (ARIN/.APNIC) like you mentioned above,  dont remember the frequency of the updates.   
Not sure this is still a thing in 2.3

**Comment by weslambert** [comment]
Yes, it is sounds like GeoIP update activity.

**Comment by dougburks** [comment, marked as answer]
It seems like you're still using the old Security Onion 16.04. If that is the case, please note that 16.04 is past End Of Life and you should make plans to upgrade to the current Security Onion 2.3:
https://docs.securityonion.net/en/2.3/eol.html
